### PR TITLE
watchdog fixes

### PR DESCRIPTION
**Pull request by deepthiskumar**
1. specify container names for the recent-logs metric
2. fix reachable seeds metric. Exposing libp2p errors in the case of failures.
Seeing these libp2p errors locally. 
```
1. failed to dial 12D3KooWJ44yJKh9PtJSoS6GASFkz5qtVboCyhiD1ZTmSV3QiSUn: all dials failed
  * [/ip4/35.245.137.202/tcp/10000] failed to negotiate security protocol: message did not have trailing newline
2. context deadline exceeded
3. failed to dial 12D3KooWJcdhZst7gHYK4QiupZgJPBAG5g6T7d7Xd6jFRxEHRrDc: all dials failed
  * [/ip4/35.221.54.181/tcp/10002] dial tcp4 0.0.0.0:43383->35.221.54.181:10002: i/o timeout
```

### CODE REVIEW

File: automation/services/watchdog/check_libp2p/check_libp2p.go [hunk 1]
```diff
@@ -31,11 +31,13 @@ func main() {
 	for i, addr := range addrs {
 		maddr, err := ma.NewMultiaddr(addr)
 		if err != nil {
+			fmt.Fprintln(os.Stderr,err)
```
Comment (imeckler) [inline automation/services/watchdog/check_libp2p/check_libp2p.go:34]: Been talking to @noot about this -- she thinks the failures may have something to do with the chain id not being set in this program.

File: automation/services/watchdog/metrics.py [hunk 2]
```diff
@@ -103,13 +103,19 @@ def check_seed_list_up(v1, namespace, seeds_reachable):
...
+        print("check_libp2p error: {}".format(line))
+  val = proc.stdout.read()
+  print("check_libp2p output: {}".format(val))
+  proc.stdout.close()
+  proc.wait()
+
+  res = json.loads(val)
+  #checklibp2p returns whether or not the connection to a peerID errored
+  successful_connections = list(filter(None,res.values()))
+  seeds_reachable.set(len(successful_connections))
```
Comment (imeckler) [inline automation/services/watchdog/metrics.py:119]: Since this is no longer a fraction, do we need to edit the limit on the corresponding alert?

Comment (deepthiskumar) [reply automation/services/watchdog/metrics.py:119]: Actually, I'll change it back to a fraction since the seed peer list for a network can be updated

**Comment by imeckler** [comment]
!approved-for-mainnet

**Pull request merged**